### PR TITLE
move relevant info from playbook readme to root readme

### PR DESCRIPTION
**Pull request by des-des**
related #42

### CODE REVIEW

File: playbook/README.md [hunk 1]
```diff
@@ -1,12 +1,3 @@
-#Founders &amp; Coders Playbook [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/foundersandcoders/playbook/issues)
-
-This playbook repo is intended to be a reference on how we do things at [Founders &amp; Coders](http://www.foundersandcoders.org).
-
-**All pull requests welcome** (remember to update the [contents section](#contents) of this readme as part of your PR).
```
Comment (iteles) [inline playbook/README.md]: Is this part not worth adding in, with a change to the reviewers? (Although we should note the requirement for an issue first)

Comment (bradreeder) [inline playbook/README.md]: I agree, I'd add it back in with me, you, Ines, Sofer & Nelson as reviewers?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
-# master-reference
-A master reference for the running of F&amp;C, including the curriculum
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/foundersandcoders/master-reference/issues)
+
+# Founders &amp; Coders Playbook 
+
+A master reference for the running of [Founders &amp; Coders](http://www.foundersandcoders.org), including the curriculum.
+
+**All pull requests welcome** (remember to update the [contents section](#contents) of this readme as part of your PR).
```
Comment (bradreeder) [inline README.md]: We should probably have a line on the necessity of opening an issue, if their issue doesn't already exist, before making a PR, as @iteles said.

Contents section link currently doesn't link anywhere but I think this is fine as we should probably add a contents section later on?

Comment (des-des) [inline README.md]: Ha, was making this as you were reviewing: #54.

**Comment by des-des** [comment]
@bradreeder I have made this issue: #54. Do you want to
 1. Provisionally add those points in the readme
 2. Leave that out of this pull.
 3. Wait for discussion of #54 then come back to this pull.

**Comment by bradreeder** [comment]
@des-des I'm easy on these options. If #54 is something in the works short/mid-term we may as well wait I guess?

It's potentially relevant in that I was going to go through the workflow we've been using with F&C9 tomorrow so they know how to commit their work here. Do you want me to use the outline you've suggested in #54 ?

File: README.md (hunk 2 above)
Comment (bradreeder) [inline README.md]: Should we just change this line to read: 'All pull requests welcome. Remember to first consult our contributing guide.' ?

**Comment by des-des** [comment]
yup we are thinking in parallel (great minds)!

File: README.md [hunk 3]
```diff
@@ -1,9 +1,7 @@
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/foundersandcoders/master-reference/issues)
 
-# Founders &amp; Coders Playbook 
+# Founders &amp; Coders Playbook
```
Comment (bradreeder) [inline README.md]: We should probably change the header's title from 'Founders & Coders Playbook' to 'Founders & Coders' as the playbook is out of date?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,12 +1,14 @@
 Inspired by [dwys's contribution workflow](https://www.github.com/dwyl/contributing).
```
Comment (bradreeder) [inline CONTRIBUTING.md]: Didn't catch this one before. 'dwyl' rather than 'dwys'

'Do what you say'? 😸

**Pull request merged**